charts/loki-simple-scalable/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: loki-simple-scalable
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.5.0
-version: 1.1.0
+version: 1.2.0
```

Comment:
WDYT of adding a changelog file to this chart?

Reply:
I think that's a great idea!